scripts/Interface/ui/drill/model/specifications.py

```diff
@@ -0,0 +1,210 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+
+class RundexSettings(object):
+
+    # instruments
+    D11 =    "D11"
+    D16 =    "D16"
+    D22 =    "D22"
+    D33 =    "D33"
+    D17 =    "D17"
+    FIGARO = "FIGARO"
+
+    # techniques
+    SANS = "SANS"
+    REFL = "Reflectometry"
+
+    # acquisition modes
+    SANS_ACQ =   "SANS"
+    REFL_POL =   "Polarized"
+    REFL_NPOL =  "Unpolarized"
+
+    # correspondance between instrument and technique
+    TECHNIQUE = {
+            D11:    SANS,
+            D16:    SANS,
+            D22:    SANS,
+            D33:    SANS,
+            D17:    REFL,
+            FIGARO: REFL
+            }
+
+    # correspondance between instrument and acquisition mode
+    ACQUISITION_MODES = {
+            D11:    [SANS_ACQ],
+            D16:    [SANS_ACQ],
+            D22:    [SANS_ACQ],
+            D33:    [SANS_ACQ],
+            D17:    [REFL_POL, REFL_NPOL],
+            FIGARO: [REFL_NPOL]
+            }
+
+    # parameters for each acquisition mode
+    COLUMNS = {
+            SANS_ACQ: [
+                "SampleRuns",
+                "SampleTransmissionRuns",
+                "AbsorberRuns",
+                "BeamRuns",
+                "FluxRuns",
+                "ContainerRuns",
+                "ContainerTransmissionRuns",
+                "TransmissionAbsorberRuns",
+                "TransmissionBeamRuns",
+                "MaskFiles",
+                "ReferenceFiles",
+                "OutputWorkspace",
+                "WedgeWorkspace",
+                "SampleThickness",
+                "CustomOptions"
+                ],
+            REFL_POL: [
+                "Run00",
+                "Run01",
+                "Run10",
+                "Run11",
+                "DirectRun",
+                "OutputWorkspace",
+                "AngleOption",
+                "Theta",
+                "SummationType",
+                "CustomOptions",
+                "GlobalScaleFactor",
+                "UseManualScaleFactors",
+                "ManualScaleFactors"
+                ],
+            REFL_NPOL: [
+                "Run",
```

Comment:
Make the NPOL and POL consistent in the column order; the only difference is the runs.

Reply:
I change the order. They are now consistent.